aws/dms/tasks.yml

```diff
@@ -20,8 +20,6 @@ cron-pii:
 - dashboard.authentication_options:
     remove_column: [data]
 - dashboard.peer_reviews
-- dashboard.circuit_playground_discount_applications
-- dashboard.circuit_playground_discount_codes
```

Comment:
Checking [here](https://codedotorg.slack.com/archives/C03CK49G9/p1702672172631009) with infra to see if editing this file is okay and if it makes sense for it to be in this PR (rather than the followup PR that actually drops the `CircuitPlaygroundDiscountApplication` and `CircuitPlaygroundDiscountCode` tables)